ui/components/multichain-accounts/add-multichain-account/add-multichain-account.test.tsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
-import { fireEvent, screen } from '@testing-library/react';
+import { fireEvent, screen, waitFor } from '@testing-library/react';
 import { renderWithProvider } from '../../../../test/lib/render-helpers';
 import configureStore, { MetaMaskReduxDispatch } from '../../../store/store';
 import { createNextMultichainAccountGroup } from '../../../store/actions';
 import { AddMultichainAccount } from './add-multichain-account';
+import { useAccountsOperationsLoadingStates } from '../../../hooks/accounts/useAccountsOperationsLoadingStates';
```

Comment:
Lint issue here: `../../../hooks/accounts/useAccountsOperationsLoadingStates import should occur before import of ./add-multichain-account`